templates/tpos/index.html

```diff
@@ -591,6 +591,7 @@ <h6 class="text-subtitle1 q-my-none">
       return {
         tposs: [],
         currencyOptions: [
+          'satoshi',
```

Comment:
Normally `sats` it used for `satoshis`. 
See env var here https://github.com/lnbits/lnbits/blob/dev/.env.example#L70

```env
LNBITS_DENOMINATION=sats
```